guide-sharding-and-replication.md

```diff
@@ -105,13 +112,13 @@ AS SELECT
     service,
     operation,
 count() as count
-FROM jaeger_index_local
+FROM jaeger.jaeger_index_local
```

Comment:
This is unfortunate that the database has to be explicitly specified even though `use jaeger` has been run before. If the DB is not specified the `default` is used.

Isn't there a way to specify "current" database ?